src/components/Blocking/ReportPostAnonymous.js

```diff
@@ -119,7 +119,7 @@ const styles = StyleSheet.create({
     marginBottom: 30
   },
   btnSkip: {
-    backgroundColor: COLORS.lightgrey,
+    backgroundColor: COLORS.gray100,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The background color change from `COLORS.lightgrey` to `COLORS.gray100` is acceptable as long as it fits the new design guidelines. Please verify that this color change does not affect readability or usability.
```diff
-    backgroundColor: COLORS.lightgrey,
+    backgroundColor: COLORS.gray100,
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->